src/test/scala/exchangeapi/NodesSuite.scala

```diff
@@ -1103,8 +1106,8 @@ class NodesSuite extends FunSuite {
     assert(response.code === HttpCode.POST_OK)
     val postResp = parse(response.body).extract[PostNodeErrorResponse]
     assert(postResp.nodes.size === 2)
-    assert(postResp.nodes.head === "NodesSuiteTests/n1")
-    assert(postResp.nodes(1) === "NodesSuiteTests/n2")
+    assert(postResp.nodes.contains("NodesSuiteTests/n1"))
+    assert(postResp.nodes.contains("NodesSuiteTests/n2"))
```

Comment:
Thx for fixing this. I actually hit this once yesterday when running the test suites, but didn't have time to investigate, and reran the test suites and then everything worked (because it randomly depended on what order they came back in), so i moved on. But it left a nagging concern, but now that concern is gone :)